### PR TITLE
Single persistent dock

### PR DESCRIPTION
**Pull request by tokejepsen**

### CODE REVIEW

File: pyblish_maya/lib.py [hunk 1]
```diff
@@ -331,10 +330,17 @@ def dock(window):
...
-                     content=dock.objectName(), allowedArea=["right", "left"])
+    dock_control = cmds.dockControl(label=window.windowTitle(), area="right",
+                                    visible=True, content=dock.objectName(),
+                                    allowedArea=["right", "left"])
 
     dock.layout().addWidget(window)
+
+    # delete any existing dock control
+    if self._dock_control:
+        if cmds.dockControl(self._dock_control, q=True, ex=True):
```
Comment (mottosso) [inline pyblish_maya/lib.py]: For our future selves, when we've forgotten the details of `dockControl`, could you use the fully qualified names of arguments? E.g. I'm assuming `q=True` means `query=True`.

Comment (tokejepsen) [inline pyblish_maya/lib.py]: Very good point.

**Comment by tokejepsen** [comment]
Just realised that I wasn't using the single persistent pyblish-lite window, so this is not ready.

**Comment by tokejepsen** [comment]
Now it works :)

**Comment by tokejepsen** [comment]
Consider this ready for merge.

**Comment by mottosso** [comment]
Looks good, merge at will!

**Comment by tokejepsen** [comment]
https://github.com/pyblish/pyblish-maya/releases/tag/2.1.2

**Pull request merged**